Functional/functional.py

```diff
@@ -0,0 +1,93 @@
+
+"""
+sequential_map - функция должна принимать в качестве аргументов любое количество функций
+(позиционными аргументами, НЕ списком), а также контейнер с какими-то значениями.
+Функция должна возвращать список результатов последовательного применения переданных функций
+к значениям в контейнере. Например,
+sequential_map(np.square, np.sqrt, lambda x: x**3, [1, 2, 3, 4, 5]) -> [1, 8, 27, 64, 125]
+"""
+
+
+def sequential_map(*args):
+    *functions, values_ = args
+    results = values_
+    for func in functions:
+        results = func(results)
+    return results
+
+
+""" 
+consensus_filter - функция должна принимать в качестве аргументов любое количество функций 
+(позиционными  аргументами, НЕ списком), возвращающих True или False, а также контейнер с какими-то значениями. 
+Функция должна возвращать список значений, которые при передаче их во все функции дают True. 
+Например: consensus_filter(lambda x: x > 0, lambda x: x > 5, lambda x: x < 10, [-2, 0, 4, 6, 11]) -> [6]
+"""
+
+
+def consensus_filter(*args):
+    *functions, values_ = args
+    for func in functions:
+        if values_:
+            passed = list(map(func, values_))
+            values_ = [x for i, x in enumerate(values_) if passed[i]]
+        else:
+            break
```

Comment:
Поздравляю, у вас велосипед)
```suggestion
        values_ = filter(func, values_)
```